src/tag-input/src/TagInput.js

```diff
@@ -151,7 +151,7 @@ const TagInput = memo(
 
     const { tokens } = useTheme()
     const themedContainerClassName = useTagInputAppearance()
-    const themedInputClassName = useInputAppearance('none')
+    const { className: themedInputClassName, boxProps } = useInputAppearance({ appearance: 'none' })
```

Comment:
If there's only one appearance, should this just be the base style? (No modifiers?)

Reply:
This is an actual variant of the component. Maybe we could rename it to something else to make it more clear what it is

Reply:
Ah gotcha! Seems like it's not much of a variant if this is the only value we pass in! But I guess this is because we are reusing the same hook in multiple components?

Reply:
Since we switched this to use the actual `<TextInput />` component this goes away :)